x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/expression/function/AbstractAggregationTestCase.java

```diff
@@ -150,8 +151,10 @@ private void aggregateSingleMode(Expression expression) {
         Object result;
         try (var aggregator = aggregator(expression, initialInputChannels(), AggregatorMode.SINGLE)) {
             for (Page inputPage : rows(testCase.getMultiRowFields())) {
-                try {
-                    aggregator.processPage(inputPage);
+                try (
+                    BooleanVector noMasking = driverContext().blockFactory().newConstantBooleanVector(true, inputPage.getPositionCount())
```

Comment:
At the point of this PR, we should be able to test masking here. Maybe making another test for it.
Should we do it now? Or in other PR?

Reply:
Yeah! I was going to do it in a follow-up. But, yeah. Soon!